report_wkhtmltopdf_param/models/report_paperformat.py

```diff
@@ -36,7 +36,8 @@ def _check_recursion(self):
             report = self.env["ir.actions.report"].new(
                 {"paperformat_id": paperformat.id}
             )
-            content = report._run_wkhtmltopdf(sample_html)
+            contenthtml = [bytes(sample_html, "utf-8")]
```

Comment:
Test now fails on the `bytes` object.... maybe it should be just `[sample_html]` ?

Reply:
@thomaspaulb think we good now :smiling_face_with_tear:

Reply:
Yayyy green